lms/djangoapps/grades/scores.py

```diff
@@ -21,12 +21,34 @@ def block_types_possibly_scored():
     since those children might have scores. We can avoid things like Videos,
     which have state but cannot ever impact someone's grade.
     """
-    return frozenset(
-        cat for (cat, xblock_class) in XBlock.load_classes() if (
-            getattr(xblock_class, 'has_score', False) or getattr(xblock_class, 'has_children', False)
-        )
```

Comment:
This file doesn't look right at all, I think the git merge got applied strangely.